### ISSUE TITLE
Uncaught ReferenceError: _this is not defined

### ISSUE THREAD

**Issue by vimniky** [EXSyst/recent-projects#43]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.7.3
**System**: Mac OS X 10.11.4
**Thrown From**: [recent-projects](https://github.com/EXSyst/recent-projects) package, v0.11.1

### Stack Trace

Uncaught ReferenceError: _this is not defined

```
At /Users/vimniky/.atom/packages/recent-projects/lib/recent-projects-view.js:245

ReferenceError: _this is not defined
    at /Users/vimniky/.atom/packages/recent-projects/lib/recent-projects-view.js:245:5
    at RecentProjectsView.selectPosition (/Users/vimniky/.atom/packages/recent-projects/lib/recent-projects-view.js:194:4)
    at RecentProjectsView.moveRight (/Users/vimniky/.atom/packages/recent-projects/lib/recent-projects-view.js:244:9)
    at /Users/vimniky/.atom/packages/recent-projects/lib/recent-projects-view.js:417:11
    at write (/Users/vimniky/.atom/packages/recent-projects/lib/recent-projects-manager.js:50:10)
    at /Users/vimniky/.atom/packages/recent-projects/lib/recent-projects-manager.js:62:11
    at read (/Users/vimniky/.atom/packages/recent-projects/lib/recent-projects-manager.js:36:9)
    at alter (/Users/vimniky/.atom/packages/recent-projects/lib/recent-projects-manager.js:55:9)
    at Object.remove (/Users/vimniky/.atom/packages/recent-projects/lib/recent-projects-manager.js:142:10)
    at RecentProjectsView.removeProject (/Users/vimniky/.atom/packages/recent-projects/lib/recent-projects-view.js:415:32)
    at HTMLButtonElement.<anonymous> (/Users/vimniky/.atom/packages/recent-projects/lib/recent-projects-view.js:391:19)
    at HTMLButtonElement.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
    at HTMLButtonElement.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)
```

### Commands

```
     -0:07.4.0 atom-hide-tabs:toggle (atom-workspace.workspace.scrollbars-visible-when-scrolling.seti-ui-no-icons)
     -0:03.4.0 core:copy (atom-workspace.workspace.scrollbars-visible-when-scrolling.seti-ui-no-icons.theme-one-dark-syntax.theme-atom-dark-ui)
```

### Config

```json
{
  "core": {
    "customFileTypes": {
      "source.ini": [
        ".buckconfig",
        ".flowconfig"
      ],
      "source.json": [
        ".arcconfig",
        "BUCK.autodeps"
      ],
      "source.python": [
        "BUCK"
      ]
    },
    "disabledPackages": [
      "vim-modeline",
      "vim-mode-plus",
      "tool-bar",
      "autocomplete-haskell",
      "haskell-ghc-mod",
      "haskell-grammar",
      "ide-haskell",
      "ide-haskell-cabal",
      "ide-haskell-repl",
      "language-haskell",
      "file-type-icons",
      "linter-flow",
      "git-diff-details",
      "git-control",
      "nuclide",
      "symbols-tree-view",
      "markdown-preview-plus",
      "gist-it",
      "tree-view-git-branch",
      "an-color-picker",
      "symbols-view",
      "git-projects",
      "open-recent"
    ],
    "packagesWithKeymapsDisabled": [
      "git-diff-details",
      "atom-ternjs",
      "git-history",
      "terminal-plus",
      "markdown-preview"
    ],
    "themes": [
      "atom-dark-ui",
      "one-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
activate-power-mode, v0.5.2
atom-alignment, v0.12.1
atom-ctags, v4.5.2
atom-hide-tabs, v0.2.1
atom-html-preview, v0.1.19
atom-react-native-css, v1.1.0
atom-ternjs, v0.14.1
autoclose-html, v0.23.0
autocomplete-ctags, v0.4.1
autocomplete-html-entities, v0.1.0
autocomplete-paths, v1.0.2
clipboard-plus, v0.5.1
color-picker, v2.1.1
docblockr, v0.7.3
emmet, v2.4.3
file-icons, v1.7.10
filecolor, v0.3.1
git-log, v0.4.1
language-babel, v2.19.4
language-docker, v1.1.6
language-javascript-jsx, v0.3.7
linter, v1.11.4
markdown-scroll-sync, v2.1.2
minimap, v4.23.5
pdf-view, v0.50.0
pigments, v0.26.2
pretty-json, v1.0.3
react, v0.15.0
react-es6-snippets, v0.3.0
react-snippets, v0.4.2
recent-projects, v0.11.1
term3, v0.22.0
terminal-plus, v0.14.5
tree-view-git-status, v0.2.3
vim-mode, v0.65.0
vim-mode-clipboard-plus, v0.2.0
vim-mode-visual-block, v0.2.15
vim-surround, v0.8.1

# Dev
No dev packages
```

**Comment by vimniky** [comment]
I can't disable Recent-project ! And when I invoke the  Recent-project comand  it just restart atom again and again !

**Comment by Ener-Getick** [comment]
I'm sorry i won't be able to take a look at this before tomorrow...
At worst you can temporarilly uninstall it by running ``apm remove recent-projects``.

**Comment by Ener-Getick** [comment]
Fixed in ``0.11.2`` thanks !
